examples/flows/chat/chat-with-pdf/tests/azure_chat_with_pdf_test.py

```diff
@@ -82,9 +82,8 @@ def test_bulk_run_mapping_missing_one_column(self):
         )
         self.pf.stream(run)  # wait for completion
 
-        self.assertEqual(run.status, "Failed")
-        with self.assertRaises(Exception):
-            _ = self.pf.get_details(run)
+        # run won't be failed, only line runs inside it will be failed.
+        self.assertEqual(run.status, "Completed")
```

Comment:
Could we assert on line run result then?

Reply:
Seems we don't have the line run results for pfazure run.

Reply:
offline synced, added a todo